pkg/haprovider/haprovider.go

```diff
@@ -138,15 +138,16 @@ func (r *HAProvider) annotateService(ctx context.Context, cluster *clusterv1.Clu
 		return serviceAnnotation, nil
 	}
 
-	aviInfraSetting, err := r.getAviInfraSettingFromCluster(ctx, cluster)
-	if err != nil {
-		return serviceAnnotation, err
-	}
+	// Temporarily commenting adding annotation to HA svc, will uncomment it after the feature is fully tested.
```

Comment:
Could you add //TODO as well

Reply:
Is there any integration test case about this

Reply:
+1 on using TODO e.g. `@TODO(lxinqi)`

Reply:
No, currently we don't have any intg test about this. Will add intg test cases in the code orange.